service-proxy.js

```diff
@@ -1400,8 +1400,9 @@ function reapSinglePeer(hostPort, serviceNames, now) {
         peer.clearDrain('superceded by peer reap');
     }
 
-    for (var i = 0; i < serviceNames.length; i++) {
-        var serviceName = serviceNames[i];
+    var serviceKeys = Object.keys(serviceNames);
```

Comment:
more minimal to rename the arg and then have `var serviceNames = Object.keys(someArg)`... the values are what, last refresh / advertise time? is it useful to log those too?